<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-PQ7IP8FG" failures="1" tests="1" name="cucumber.TestNGTestRunner" time="8.392" errors="0" timestamp="2024-08-03T03:37:44 IST" skipped="0">
  <testcase classname="cucumber.TestNGTestRunner" name="runScenario" time="8.392">
    <failure message="expected:&lt;THANKYOU FOR THE ORDER.&gt; but was:&lt;Thankyou for the order.&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<THANKYOU FOR THE ORDER.> but was:<Thankyou for the order.>
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:115)
at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:125)
at saloni.stepDefinition.stepDefinitionimpl.message_is_displayed_on_confirmation_page(stepDefinitionimpl.java:59)
at âœ½."Thankyou for the order." message is displayed on confirmationPage(file:///C:/Users/saloni%20jain/eclipse-workspace/projectfarmework/src/test/java/cucumber/PurchaseOrder.feature:32)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- cucumber.TestNGTestRunner -->
